tox.ini

```diff
@@ -2,16 +2,19 @@
 envlist =
        ; disable creating docs until https://github.com/mkdocs/mkdocs/issues/807 is resolved
        ; py34-{flake8,docs}
```

Comment:
before we enable them, we should test that locally - I'm not even sure they still work.